Homework2: Andrei Bartosh

sharp-0000ff opened the pull request.

sharp-0000ff committed:
release 0.1

sharp-0000ff committed:
Homework2: Andrei Bartosh

sharp-0000ff committed:
release 0.2

sharp-0000ff committed:
release 0.2

sharp-0000ff committed:
release 0.2

sharp-0000ff committed:
release 0.2

sharp-0000ff committed:
release 0.2

belovmd reviewed (changes requested):
Thank you for your work. 
Please address next paragraphs from review notes [Review Notes](https://github.com/belovmd/it-academy-python-spring/blob/master/homework_review/Homework2.md):
- 1
- 2
- 3
- 4

sharp-0000ff committed:
Make fixes after Code Review
Used % operator instead of while
Check on punctuation marks
Remove check for != ' '
Added advanced tasks

sharp-0000ff committed:
Make fixes after Code Review
Used % operator instead of while
Check on punctuation marks
Remove check for != ' '
Added advanced tasks

sharp-0000ff committed:
Make fixes after Code Review
Used % operator instead of while
Check on punctuation marks
Remove check for != ' '
Added advanced tasks

sharp-0000ff committed:
added description advanced tasks

belovmd commented:
great commit messages but they are duplicated for all comnits :)

belovmd reviewed (approved):
8
tasks don't look as advanced
